EventFilter/L1TXRawToDigi/plugins/L1TCaloLayer1RawToDigi.cc

```diff
@@ -78,13 +78,13 @@ class L1TCaloLayer1RawToDigi : public stream::EDProducer<> {
   virtual void produce(Event&, const EventSetup&) override;
   virtual void endStream() override;
 
-  void makeECalTPGs(uint32_t lPhi, UCTCTP7RawData& ctp7Data, std::unique_ptr<EcalTrigPrimDigiCollection>& ecalTPGs);
+  void makeECalTPGs(uint32_t lPhi, UCTCTP7RawData& ctp7Data, std::auto_ptr<EcalTrigPrimDigiCollection>& ecalTPGs);
```

Comment:
These should be unique_ptr, please update so this PR does not change unique_ptr.
I guess you directly copied this file from 80x, from before auto_ptr -> unique_ptr migration happened.